lib/KListWithOverflow.vue

```diff
@@ -195,11 +207,17 @@
           maxWidth -= removedDividerWidth;
         }
 
-        maxWidth = Math.ceil(maxWidth);
-        this.overflowItems = overflowItemsIdx.map(idx => this.items[idx]);
+        this.overflowItems = overflowItemsIdx.map((idx) => this.items[idx]);
         this.isMoreButtonVisible = overflowItemsIdx.length > 0;
-        list.style.maxWidth = `${maxWidth}px`;
+        list.style.maxWidth = `${Math.ceil(maxWidth)}px`;
         list.style.maxHeight = `${maxHeight}px`;
+
+        this.setVisibleItems(directionIndexes, overflowItemsIdx);
```

Comment:
We are not doing anything with this `visibleItems` array. I would suggest to remove them. Also I dont note any added value of removing the `maxWidth = Math.ceil(maxWidth);` sentence. I think we can keep these lines as before:
```javascript
        maxWidth = Math.ceil(maxWidth);
        this.overflowItems = overflowItemsIdx.map(idx => this.items[idx]);
        this.isMoreButtonVisible = overflowItemsIdx.length > 0;
        list.style.maxWidth = `${maxWidth}px`;
        list.style.maxHeight = `${maxHeight}px`;
      },
```